src/index.js

```diff
@@ -232,6 +232,8 @@ class TerserPlugin {
             if (this.options.cache) {
               const defaultCacheKeys = {
                 terser: terserPackageJson.version,
+                'terser_no_bundle_env': process.env.TERSER_NO_BUNDLE,
+                'node_version': process.version,
```

Comment:
Can `terser` change output based on `nodejs` version, right?

Reply:
/cc @fabiosantoscode can find logic https://github.com/terser-js/terser/search?p=1&q=process.version&unscoped_q=process.version, can you provide link on source code where you use `process.version`?

Reply:
Given some bug, it might just change behaviour

Reply:
And there's some regex details and other small things that change due to added and removed variables in the global objects.

Reply:
We don't use process.version but the output still changes.

Reply:
Because the node version is different and there are small differences that are affected by it

Reply:
@fabiosantoscode can you provide example with pure `terser` package?

Reply:
I wanted to reply, but didn't "submit" the review. I still don't know how to use these. Here's the example:

```
fabio@fabio-thinkpad ♥  nvm use 6
Now using node v6.14.3 (npm v3.10.10)
fabio@fabio-thinkpad ♥  echo '/with dotall flag/s' | terser -mc
Parse error at 0:1,0
/with dotall flag/s
^
ERROR: Invalid flags supplied to RegExp constructor 's'
    at JS_Parse_Error.get (/home/fabio/devel/terser/lib/utils.js:83:23)
    at fatal (/home/fabio/devel/terser/bin/uglifyjs:308:52)
    at run (/home/fabio/devel/terser/bin/uglifyjs:247:9)
    at Socket.<anonymous> (/home/fabio/devel/terser/bin/uglifyjs:183:9)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:978:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

Reply:
:+1: